### PR TITLE
Added Stimulus JS Controllers

### PR DESCRIPTION
**Pull request by alexisKinto**
Closes: https://github.com/Grupo-AFAL/bali-view-components/issues/90
Added the following Stimulus JS Controllers.
- auto-play-audio
- autocomplete-address
- elements-overlap
- focus-on-connect
- input-on-change
- print
- radio-toggle
- submit-button
- checkbox-toggle

Added TODOs with Issue #, for future testing of this controllers
Fixed typo in a test

### CODE REVIEW

**Review by rafaelcdelgado — APPROVED** [review]
Minor recommendations..

File: app/javascript/bali/index.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import GoogleMapsLoader from './utils/google-maps-loader'
+import * as time from './utils/time'
```
[the remark this answers is not in the document]
Comment (alexisKinto) [reply app/javascript/bali/index.js:2]: Yes we do

File: app/javascript/bali/controllers/submit-button-controller.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * SubmitButton Controller
+ * Displays a loading button state when a form submission is started
+ * It relies on @hotwired/turbo events to display and hide the loading state
+ *
+ * https://bulma.io/documentation/elements/button/#states
+ */
+export class SubmitButtonController extends Controller {
+  connect () {
+    this.element.addEventListener('turbo:submit-start', e => {
+      this.disableButton(e.detail.formSubmission.submitter)
```
[the remark this answers is not in the document]
Comment (alexisKinto) [reply app/javascript/bali/controllers/submit-button-controller.js:13]: We should refactor this controllers when we add their proper tests

[the remark this answers is not in the document]
Comment (rafaelcdelgado) [reply app/javascript/bali/controllers/submit-button-controller.js:13]: @alexisKinto , totally agred!

**Review by fedegl — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.0] - 2022-06-27
```
Comment (Ivan96318) [inline CHANGELOG.md:8]: Today is June 28th not 27th 😅😬

**Review by Ivan96318 — APPROVED** [review]

**Pull request merged**